cmd/convert.go

```diff
@@ -35,15 +36,32 @@ can be converted into a 'konnect' configuration file.`,
 				return err
 			}
 
-			if yes, err := utils.ConfirmFileOverwrite(convertCmdOutputFile, "", false); err != nil {
-				return err
-			} else if !yes {
-				return nil
-			}
+			if convertCmdInputFile != "" {
+				if yes, err := utils.ConfirmFileOverwrite(convertCmdOutputFile, "", false); err != nil {
+					return err
+				} else if !yes {
+					return nil
+				}
 
-			err = convert.Convert(convertCmdInputFile, convertCmdOutputFile, sourceFormat, destinationFormat)
-			if err != nil {
-				return fmt.Errorf("converting file: %v", err)
+				err = convert.Convert(convertCmdInputFile, convertCmdOutputFile, sourceFormat, destinationFormat)
+				if err != nil {
+					return fmt.Errorf("converting file: %v", err)
+				}
+			} else {
+				path, err := os.Getwd()
```

Comment:
This should execute only when source and destination format are 2.x and 3.x respectively.

Reply:
https://github.com/Kong/deck/pull/753/commits/6347de40814d87c4ad9d9365b01202b315c640c4